spec/recipes/mistral_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe 'stackstorm::mistral' do
+  before do
+    global_stubs_include_recipe
+  end
+
+  context 'with default node attributes' do
+    let(:chef_run) { ChefSpec::SoloRunner.new.converge(described_recipe) }
+
+    it 'should include recipe openstack-mistral' do
+      allow_any_instance_of(Chef::Recipe).to receive(:include_recipe).with('openstack-mistral')
+      chef_run
+    end
+
+    it 'should create mysql_service "default"' do
```

Comment:
Gotcha, @Kami approved the last major change where this was missed.
Going to the leave the spec as is since thats what the recipe does right now